src/auth0/handlers/connections.js

```diff
@@ -44,7 +44,22 @@ export default class ConnectionsHandler extends DefaultHandler {
     if (this.existing) return this.existing;
     const connections = await this.client.connections.getAll({ paginate: true });
     // Filter out database connections
-    return connections.filter(c => c.strategy !== 'auth0');
+    this.existing = connections.filter(c => c.strategy !== 'auth0');
+
+    // Convert enabled_clients from id to name
+    const clients = await this.client.clients.getAll({ paginate: true });
+    this.existing = this.existing.map(connection => ({
+      ...connection,
+      enabled_clients: [
+        ...(connection.enabled_clients || []).map((clientId) => {
+          const found = clients.filter(c => c.client_id === clientId)[0];
```

Comment:
might be a little cleaner to use [`find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) here instead of `filter()[0]`

Also, feels like this function could be pulled out and reused for `clientGrants` and `databases`, maybe into the utils file? But that might be overkill...

```
export function getClientName(clients, clientId) {
  const found = clients.find(c => c.client_id === clientId);
  if (found) return found.name;
  return clientId;
}
```

Reply:
fixed. Didn't add common method as it seemed overkill to me.